commands.c

```diff
@@ -1424,18 +1431,6 @@ void commands_apply_mcconf_hw_limits(mc_configuration *mcconf) {
     }
 
 #ifndef DISABLE_HW_LIMITS
-
```

Comment:
This comment should still be there.

Reply:
uh, do i change something there ? that was an accident